### PR TITLE
s/buton/button

### PR DESCRIPTION
**Pull request by PhilETaylor**
s/buton/button

### CODE REVIEW

**Comment by Twincarb** [comment]
I have tested this item :white_check_mark: successfully on 474e3085027d0f49cdba4834ad642f8b56b7c0f9<br /><br />Code Review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23007">issues.joomla.org/tracker/joomla-cms/23007</a>.</sub>

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 474e3085027d0f49cdba4834ad642f8b56b7c0f9<br /><br />Code review + check that this PR covers all occurrences of wrong English "buton" (case-insensitive) in Joomla 4.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23007">issues.joomla.org/tracker/joomla-cms/23007</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23007">issues.joomla.org/tracker/joomla-cms/23007</a>.</sub>

File: templates/system/offline.php [hunk 1]
```diff
@@ -68,7 +68,7 @@
 				<input type="text" name="secretkey" class="inputbox" alt="<?php echo JText::_('JGLOBAL_SECRETKEY'); ?>" id="secretkey" />
 			</p>
 		<?php endif; ?>
-		<p id="submit-buton">
+		<p id="submit-button">
```
Comment (laoneo) [inline templates/system/offline.php:71]: Did you guys check if there is no JS or CSS code related to this selector?

Comment (Twincarb) [reply templates/system/offline.php:71]: The only occurrence that is shown when doing a Github search is this one

Comment (laoneo) [reply templates/system/offline.php:71]: :+1:

**Pull request merged**